Gemfile.lock

```diff
@@ -1,33 +1,43 @@
 GEM
   remote: https://rubygems.org/
   specs:
```

Comment:
avoid changing lock file everytime. Only if it's needed commit this file. Like if you add new gem or update gem